### PR TITLE
[b] shows pending state after user select file to register asset

### PR DESCRIPTION
**Pull request by thuyenBitmark**

### CODE REVIEW

File: BitmarkRegistry/Controllers/Main/Properties/RegisterPropertyViewController.swift [hunk 1]
```diff
@@ -130,25 +130,32 @@ extension RegisterPropertyViewController: UIImagePickerControllerDelegate, UINav
 // MARK: - UIDocumentPickerDelegate
 extension RegisterPropertyViewController: UIDocumentPickerDelegate {
   func documentPicker(_ controller: UIDocumentPickerViewController, didPickDocumentAt url: URL) {
-    let didStartAccessing = url.startAccessingSecurityScopedResource()
-    defer {
-      if didStartAccessing {
-        url.stopAccessingSecurityScopedResource()
+    activityIndicator.startAnimating()
+    disabledScreen.isHidden = false
+    UIApplication.shared.beginIgnoringInteractionEvents()
```
Comment (anhnguyenbitmark) [inline BitmarkRegistry/Controllers/Main/Properties/RegisterPropertyViewController.swift]: It's deprecated? (https://developer.apple.com/documentation/uikit/uiapplication/1623047-beginignoringinteractionevents)

File: BitmarkRegistry/Controllers/Main/Properties/RegisterPropertyViewController.swift [hunk 2]
```diff
@@ -130,25 +130,32 @@ extension RegisterPropertyViewController: UIImagePickerControllerDelegate, UINav
...
+
+    DispatchQueue.main.async { [weak self] in
```
Comment (anhnguyenbitmark) [inline BitmarkRegistry/Controllers/Main/Properties/RegisterPropertyViewController.swift]: This func is on mainthread already.

Comment (thuyenBitmark) [reply BitmarkRegistry/Controllers/Main/Properties/RegisterPropertyViewController.swift]: Yes, It's on mainthread. But I call with async, so it should move to background thread and not block the mainthread executation.

File: BitmarkRegistry/Controllers/Main/Properties/RegisterPropertyViewController.swift (hunk 1 above)
Comment (thuyenBitmark) [reply BitmarkRegistry/Controllers/Main/Properties/RegisterPropertyViewController.swift]: Thanks, I will update soon.

**Pull request merged**